docker-compose.yml

```diff
@@ -24,8 +24,12 @@ boulder:
       - 9380:9380 # mail-test-srv
       - 9381:9381 # mail-test-srv
     links:
+      - bhsm:boulder-hsm
       - bmysql:boulder-mysql
       - brabbitmq:boulder-rabbitmq
+bhsm:
+    build: test/
```

Comment:
Rebuilding each time is fine, so long as it's based on another image that's cached. That said, I'd also be fine with saying we need to rebuild if test-ca.key changes (it never has so far).